modules/apps/feature-flag/feature-flag-web/src/main/java/com/liferay/feature/flag/web/internal/feature/flag/FeatureFlagsBag.java

```diff
@@ -24,10 +26,13 @@
 /**
  * @author Drew Brokke
  */
-public class CompanyFeatureFlags {
+public class FeatureFlagsBag {
```

Comment:
@thiago-buarqque , what's the reason for naming this `FeatureFlagsBag` instead of just `FeatureFlags`?

Reply:
Usually, *bag classes have a list (or map) and hold methods to retrieve info from it, parse to JSON and other methods. If we call it FeatureFlags it sounds like a simple list